data/tlg0317/tlg001/tlg0317.tlg001.1st1K-grc1.xml

```diff
@@ -0,0 +1,2980 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+<teiHeader xml:lang="eng">
+<fileDesc>
+<titleStmt>
+<title xml:lang="lat">Acta Joannis</title>
```

Comment:
Shouldn't there be an `<author>` tag as well?

Reply:
I am not sure that it makes sense to have an author here since the author is the same as the title, i.e., "Acta Joannis".

Reply:
The question I have is, whether there are going to be any scripts playing off the fact that for almost all files there is going to be a separate author. Would it do any harm to have both tags saying the same? But if it doesn't matter I will merge it. @sonofmun

Reply:
You're right. I will add it.